### PR TITLE
Hairpin NAT and reflection zone for VPN

### PR DESCRIPTION
**Pull request by cotosso**
If a reflection zone for hairpin nat is a vpn zone you need to specify its subnet.

### CODE REVIEW

File: port_forward.rst [hunk 1]
```diff
@@ -49,3 +49,64 @@ attempt to connect to a server using the public IP address, hairpin NAT ensures
...
+
+    uci show firewall | grep ".name='rwopenvpn'"
+
+   Example output: ::
+
+      firewall.ns_49d9f400.name='rwopenvpn'
+
+2. Set the desired network (in this case, **10.88.88.0/24**) for the **rwopenvpn** zone: ::
+ 
+    uci set firewall.ns_49d9f400.subnet=10.88.88.0/24
```
Comment (gsanchietti) [inline port_forward.rst:80]: The subnet field should be always a list
```suggestion
    uci add_list firewall.ns_49d9f400.subnet=10.88.88.0/24
```

File: port_forward.rst [hunk 2]
```diff
@@ -49,3 +49,64 @@ attempt to connect to a server using the public IP address, hairpin NAT ensures
...
+
+    uci show firewall | grep subnet
+
+   Example output: ::
+
+       firewall.ns_49d9f400.subnet='10.88.88.0/24'
+
+
+
+Add or remove multiple subnets to the vpn zone
```
Comment (gsanchietti) [inline port_forward.rst:99]:
```suggestion
Add or remove multiple subnets to the VPN zone
```

File: port_forward.rst [hunk 3]
```diff
@@ -49,3 +49,64 @@ attempt to connect to a server using the public IP address, hairpin NAT ensures
 out to the internet and then back into the local network.
 
 To enable the hairpin, enable the ``Hairpin NAT`` option and select one or more zones where the NAT loopback should be enabled.
+
+Hairpin NAT for VPN Zones
```
Comment (gsanchietti) [inline port_forward.rst:53]:
```suggestion
Hairpin NAT for VPN zones
```

**Review by gsanchietti — CHANGES_REQUESTED** [review]
One last note:
are you sure the correct CIDR to use is the network 10.88.88.0/24 and not the first IP of the network 10.88.88.1/24?

File: port_forward.rst [hunk 4]
```diff
@@ -49,3 +49,64 @@ attempt to connect to a server using the public IP address, hairpin NAT ensures
...
+-------------------------
+
+To use Hairpin NAT with VPN zones such as ipsec, openvpn, and rwopenvpn, additional configuration is necessary.
```
[the remark this answers is not in the document]
Comment (cotosso) [reply port_forward.rst:56]: I intentionally wrote  ipsec, openvpn, rwopenvpn  exactly as they are written in the zone section.

[the remark this answers is not in the document]
Comment (cotosso) [reply port_forward.rst:56]: About the 10.88.88.1/24 question: i made a test in a real situation and it worked flawlessy using 0 as last octet, even with 2 subnets declared for the same zone.

File: port_forward.rst [hunk 5]
```diff
@@ -49,3 +49,67 @@ attempt to connect to a server using the public IP address, hairpin NAT ensures
 out to the internet and then back into the local network.
 
 To enable the hairpin, enable the ``Hairpin NAT`` option and select one or more zones where the NAT loopback should be enabled.
+
+Hairpin NAT for VPN zones
+-------------------------
+
+To use Hairpin NAT with VPN zones such as ipsec, openvpn, and rwopenvpn, additional configuration is necessary.
```
Comment (gsanchietti) [inline port_forward.rst:56]:
```suggestion
To use Hairpin NAT with VPN zones such as `ipsec`, `openvpn`, and `rwopenvpn`, additional configuration is necessary. 
```

File: port_forward.rst [hunk 6]
```diff
@@ -49,3 +49,67 @@ attempt to connect to a server using the public IP address, hairpin NAT ensures
...
+
+To use Hairpin NAT with VPN zones such as ``ipsec`, ``openvpn``, and ``rwopenvpn``, additional configuration is necessary. 
+You must explicitly declare the subnet used by the VPN; otherwise, Hairpin NAT will not function for VPN-connected clients.
+
+This configuration can be performed via the command line. First, identify the internal reference for the zone, then add the desired network, commit the changes, and restart the service.
+
+Make sure that subnets are assigned to the correct zones:
+- `ipsec`: IPsec tunnels
+- `openvpn`: OpenVPN tunnels
+- `rwopenvpn`: OpenVPN Road Warrior
```
Comment (gsanchietti) [inline port_forward.rst:64]:
```suggestion
- ``ipsec``: IPsec tunnels
- ``openvpn``: OpenVPN tunnels
- ``rwopenvpn``: OpenVPN Road Warrior
```

**Review by gsanchietti — APPROVED** [review]

**Pull request merged**